components/Link.js

```diff
@@ -61,9 +61,17 @@ class Link extends React.Component {
       return (
         <NextLink {...pick(this.props, ['href', 'scroll'])}>
           <a
+            role="button"
+            tabIndex={0}
             className={className}
             title={title}
             onClick={onClick}
+            onKeyDown={event => {
+              if (event.key === 'Enter') {
+                event.preventDefault();
+                onClick();
+              }
+            }}
```

Comment:
This is for the `jsx-a11y/click-events-have-key-events` rule. But isn't it that we are calling `onClick` within the `onKeyDown` so it works essentially the same as `onClick` when we press enter? 🤔

Reply:
There is a regression: tab on a link, then press space. It scrolls instead of opening the link. Now that `role=button` is removed, I think that we should remove this event handler as well

![Peek 2021-03-25 09-48](https://user-images.githubusercontent.com/1556356/112445417-661c5e00-8d4f-11eb-8ff5-82f2f07a41cc.gif)

Reply:
@Betree : Ah, I've missed that. Thanks for pointing it out. I've just removed the keydown event. 👍🏽